modules/virt-metro-dr-odf.adoc

```diff
@@ -6,14 +6,11 @@
 [id="metro-dr-odf_{context}"]
 = Metro-DR for {rh-storage-first}
 
-{VirtProductName} supports the link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.14/html-single/configuring_openshift_data_foundation_disaster_recovery_for_openshift_workloads/index#metro-dr-solution[Metro-DR solution for {rh-storage}], which provides two-way synchronous data replication between managed {VirtProductName} clusters installed on primary and secondary sites. This solution combines {rh-rhacm-first}, Red{nbsp}Hat Ceph Storage, and {rh-storage} components.
+{VirtProductName} supports the link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.14/html-single/configuring_openshift_data_foundation_disaster_recovery_for_openshift_workloads/index#metro-dr-solution[Metro-DR solution for {rh-storage}], which provides two-way synchronous data replication between managed {VirtProductName} clusters installed on primary and secondary sites.
```

Comment:
In the link link I was wondering whether the 4.14 version was intentionally used. Aren't there more recent versions?

Reply:
Great point! I will update those links.